service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinClientOptions.java

```diff
@@ -0,0 +1,39 @@
+package com.microsoft.azure.sdk.iot.service.devicetwin;
+
+import com.microsoft.azure.sdk.iot.service.ProxyOptions;
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * Configurable options for all twin client operations.
+ */
+@Builder
+public class DeviceTwinClientOptions
```

Comment:
Can we not use a common client options for both methods and twins? They seem to be containing settings for the same properties.

Reply:
Right now they are the same, yes, but this separation allows us to add other options that are specific to method/twin/registry later

Reply:
Plus the one that exists right now is called "RegistryManagerOptions" so it would be a bit odd to recycle it for the twin and method clients